lib/gitmodel/index.rb

```diff
@@ -35,7 +35,7 @@ def generated?(branch = GitModel.default_branch)
     end
 
     def save(options = {})
-      GitModel.logger.debug "Saving indexes for #{@model_class}..."
+      GitModel.logger.debug 'Saving indexes for #{@model_class}...'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.